ceremonies/Cargo.toml

```diff
@@ -22,8 +22,7 @@ frame-support = { version = "4.0.0-dev", git = "https://github.com/paritytech/su
 pallet-timestamp = { version = "4.0.0-dev", git = "https://github.com/paritytech/substrate.git", default-features = false, branch = "master" }
 sp-runtime = { version = "6.0.0", git = "https://github.com/paritytech/substrate.git", default-features = false, branch = "master" }
 sp-std = { version = "4.0.0-dev", git = "https://github.com/paritytech/substrate.git", default-features = false, branch = "master" }
-
-# benchmarking
+sp-io = { version = "6.0.0", git = "https://github.com/paritytech/substrate.git", default-features = false, branch = "master" }
 frame-benchmarking = { version = "4.0.0-dev", git = "https://github.com/paritytech/substrate.git", default-features = false, optional = true, branch = "master" }
```

Comment:
Minor thing, this is not an optional dependency for benchmarking, please put in the above section, in the suitable alphabetical entry.

Also, please add `sp-io/std` in the std feature section below; otherwise this may have unforeseen effects.

Reply:
what do you mean by `the above section`?